groups/sig-release/groups.yaml

```diff
@@ -377,39 +374,36 @@ groups:
       - lauri.d.apple@gmail.com
       - saschagrunert@gmail.com
     managers:
-      - rlejano@gmail.com # 1.23 RT Lead
+      - joseph.r.sandoval@gmail.com # 1.24 Emeritus Advisor
```

Comment:
In a follow-up, let's include the 1.24 lead (@JamesLaverack)

Reply:
https://github.com/kubernetes/k8s.io/pull/3247